collectors/cgroups.plugin/cgroup-name.sh

```diff
@@ -45,11 +45,27 @@ fatal() {
   exit 1
 }
 
+function parse_docker_like_inspect_output() {
+  local output="${1}"
+  nomad_ns="$(echo "$output" | grep NOMAD_NAMESPACE | cut -d= -f 2)"
+  nomad_job_name="$(echo "$output" | grep NOMAD_JOB_NAME | cut -d= -f 2)"
+  nomad_task_name="$(echo "$output" | grep NOMAD_TASK_NAME | cut -d= -f 2)"
+  nomad_alloc_id="$(echo "$output" | grep NOMAD_SHORT_ALLOC_ID | cut -d= -f 2)"
```

Comment:
4 greps are inefficient here. Will suggest the improvement after I work on it locally.

Reply:
It is super-omega-light. The only possible improvement is fewer lines of code.

Reply:
simplified this part

Reply:
Yeah, I saw the timings, not really worth it.